### ISSUE TITLE
Please consider re-wording the commands in easier language

### ISSUE THREAD

**Issue by EmilyMalkieri** [LostPaul/obsidian-folder-notes#173]
First of all, a massive thank you for writing this plugin! Was very happy to find there was an alternative to the old plugin that I could switch to and it just immediately worked.

But please consider re-wording the command names! "Open folder note of current folder of active note?" "Make current active note a folder note for the folder of the active node?" "Create folder note with a new folder for the active note in the current folder?" That's too many notes and folders and of's and for's and too much backtracking. It borders on parody. Like, there might be a Fry & Laurie bit just like this, I wouldn't be surprised. I find it really hard to parse these sentences and I worry for people with dyslexia. And that last one's just wrong, that command creates a folder and moves our note into it. We've already got the note, we're not creating it.

I've come up with these suggestions but I'm sure together we can think of way better ones.

- "Folder notes: [Open/delete] folder note of current folder of active note" → "Folder notes: [Open/delete] this folder's linked note"
- "Folder notes: Create [type] folder note for current folder of active note" → "Folder notes: Create [a/an] [type] folder note here", perhaps with "for this folder" instead of "here" to mirror the last command
- "Folder notes: Make current active file a folder note for the folder of the active note" → "Folder notes: Use this file as the folder note"
- "Folder notes: Create folder note with a new folder for the active note in the current folder" → "Folder note: Create a folder for this note" or "Folder notes: Turn this note into a folder"

I went with "linked note" to avoid the awkward "this folder's folder note" but that's additional terminology and probably not a great idea just for that reason alone. It also might be too close to `[[wiki links]]` and `[markdown](links://)`. But even if we go with "this folder's folder note", I think just removing the constant repetition of current folders and active notes will make it much clearer.

Yes, people will have a note open when they select these commands, not a folder, but I feel like "this folder" referring to the folder that we're currently in is still plenty intuitive. Alternatively we could write a more formal "the containing folder" instead, or perhaps something very plain and casual like "[Open/delete] the folder note of the folder you're in right now."

Sincerest apologies if some of this came off too harsh, I feel like it might have.

**Comment by LostPaul** [comment]
Is this better?
![Image](https://github.com/user-attachments/assets/05e883be-e806-401a-8d9a-c6fe901b23dc)
